api/listener_test.go

```diff
@@ -0,0 +1,58 @@
+package api
+
+import (
+	"testing"
+
+	"github.com/iotexproject/iotex-core/blockchain/block"
+)
+
+var MOCKChainListener Listener
+
+// Mock ChainListener
+func TestNewChainListener(t *testing.T) {
+	t.Run("New Chain listener.", func(t *testing.T) {
```

Comment:
Because `NewChainListener` doesn't return any error, we don't really need to test it.